common/locales/en/messages.json

```diff
@@ -14,6 +15,7 @@
   "messageEquipped": " <%= itemText %> equipped.",
   "messageUnEquipped": "<%= itemText %> un-equipped.",
   "messageMissingEggPotion": "You're missing either that egg or that potion",
+  "messageInvalidEggPotionCombo": "Invalid egg+potion combination. Try hatching a different egg!",
```

Comment:
Does that only show up with quest eggs, or also of you already have the
pet? If it's just quest eggs, we can make it more descriptive:

"You can't hatch Quest Pet Eggs with Magic Hatching Potions! Try a
different egg."  That way the user understands why it isn't working.

On Wednesday, September 23, 2015, Blade Barringer <notifications@github.com>
wrote:

> In common/locales/en/messages.json
> <https://github.com/HabitRPG/habitrpg/pull/5962#discussion_r40234496>:
>
> > @@ -14,6 +15,7 @@
> >    "messageEquipped": " <%= itemText %> equipped.",
> >    "messageUnEquipped": "<%= itemText %> un-equipped.",
> >    "messageMissingEggPotion": "You're missing either that egg or that potion",
> > +  "messageInvalidEggPotionCombo": "Invalid egg+potion combination. Try hatching a different egg!",
>
> I like that better. @lemoness <https://github.com/lemoness> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/5962/files#r40234496>.
>